src/main/java/com/softserve/academy/spaced/repetition/utils/validators/PasswordNotNullValidator.java

```diff
@@ -0,0 +1,32 @@
+package com.softserve.academy.spaced.repetition.utils.validators;
+
+import com.softserve.academy.spaced.repetition.domain.Account;
+import com.softserve.academy.spaced.repetition.domain.enums.AuthenticationType;
+import com.softserve.academy.spaced.repetition.utils.validators.annotations.PasswordNotNull;
+
+import javax.validation.ConstraintValidator;
+import javax.validation.ConstraintValidatorContext;
+
+public class PasswordNotNullValidator implements ConstraintValidator<PasswordNotNull, Account> {
+
+    @Override
+    public void initialize(PasswordNotNull constraintAnnotation) {
+        /**
```

Comment:
Please move the JavaDoc out of method body. It should be writen before the method.